proto/einride/netlify/cms/v1/annotations.proto

```diff
@@ -187,6 +187,8 @@ message Field {
   bool ignore = 5;
   // Owner of the field (automatically appended to the label).
   Owner owner = 6;
+  // The summary for the list widget - see more at https://www.netlifycms.org/docs/widgets/list/.
+  string list_widget_summary = 7;
```

Comment:
This should go as a field on the list widget type instead of here.

Reply:
I see, was thinking about that and if you scroll down you can see it does have a field created there.

But the thing I had difficulty with is that I am not sure how to set the `summary` via 
annotations if we use the `ListWidget.summary` field.

With the current solution, one can set it via:
```
  repeated SomeSpec specs = 11 [(einride.netlify.cms.v1.field) = {
    list_widget_summary: "{{fields.name}} - count: {{fields.count}}"
  }];
```

But if we want to set up via `ListWidget.summary`, then the proto config would look like:
```
  repeated SomeSpec specs = 11 [(einride.netlify.cms.v1.field) = {
     widget: {
      list_widget: {
        summary: "{{fields.name}} - count: {{fields.count}}"
      }
    }
  }];
``` 

This however trigger the `// if a widget is specified, no further inference` logic (which we touched up on in the previous PR change)
https://github.com/einride/protobuf-netlify-cms/blob/24f70443a5cff412423e437800a60b8e5408e5b8/cmd/protoc-gen-netlify-cms/main.go#L412-L415

This stops the code generate other needed configurations, creating diff as:

```diff
         required: false
         hint: "A nested list of some specs to show usage of a list widget."
         widget: "list"
-        collapsed: true
-        minimize_collapsed: true
+        collapsed: false
+        minimize_collapsed: false
         summary: "{{fields.name}} - count: {{fields.count}}"
-        fields:
-
-          - name: "name"
-            label: "NAME"
-            required: false
-            widget: "string"
-            default: ""
-
-          - name: "count"
-            label: "COUNT"
-            widget: "number"
-            value_type: "int"
-            required: true
-            default: 0
```

So do you have some good idea of how we could solve it?

Reply:
So perhaps we can lose this  `// if a widget is specified, no further inference` 
to `// if a CUSTOM widget is specified, no further inference`? 

And for the standard widget case, we override the field with annotation configurations?

Or, another idea I have, is perhaps to add a field called `bool include_default_configs` on the widget type, so user can specify it as:

```
   widget: {
      include_default_configs: true
      list_widget: {
        summary: "{{fields.name}} - count: {{fields.count}}"
      }
    }
```
And the if block in the code just checks this value, instead of checking `field.Widget.WidgetType != nil`

Reply:
I made another (hopefully) better solution here: https://github.com/einride/protobuf-netlify-cms/pull/17

Reply:
Indeed.